src/main/java/core/basesyntax/services/ColorSupplier.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.services;
+
+import core.basesyntax.Colors;
+import core.basesyntax.model.Figure;
+
+public class ColorSupplier extends Figure {
```

Comment:
ColorSupplier is service
Figure is model
```suggestion
public class ColorSupplier {
```